### PR TITLE
Ci linter part 3: Lint HTML files

### PR DESCRIPTION
**Pull request by hoangviet1993**
- [x] Setup `htmllint` and `.htmlintrc` to lint HTML files.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -28,14 +28,44 @@ jobs:
           name: run tests
           command: |
             bash scripts/run_backend_tests.sh --generate_coverage_report
+    
+      - save_cache:
```
Comment (seanlip) [inline .circleci/config.yml]: What's going on here (and why are both paths the same)?

File: .circleci/config.yml [hunk 2]
```diff
@@ -28,14 +28,44 @@ jobs:
...
+      - restore_cache:
+          key: oppia-tools-cache-v1.01
+          key: node-modules-cache-v1.01
       
       - run:
           name: run linter
           command: |
             bash scripts/run_linter.sh
-
+
```
Comment (seanlip) [inline .circleci/config.yml]: drop trailing spaces

File: .circleci/config.yml (hunk 1 above)
Comment (hoangviet1993) [reply .circleci/config.yml]: So basically CircleCI encourage versioning caches. 
https://circleci.com/docs/2.0/caching/#source-caching
Due to the lack of a package.json (and therefore I cannot pipe`checksum package.json` here), I use this primitive v1.01 naming scheme. 
Whenever I add a new `install_node_module` commands, I would just change the version number and invalidate CircleCI cache. 
I figured there should be a cleaner way of doing this. Should I be `checksum`-ing the bash script file(run_linter.sh, setup.sh,etc,...)?
Yea the folder path is duplicated, will patch that asap.

Comment (seanlip) [reply .circleci/config.yml]: Chatted offline -- best course of action is probably to bring in package.json and get rid of install_node_module.

Comment (hoangviet1993) [reply .circleci/config.yml]: Yep, will replace `install_node_module` with package.json in a separate PR. Revert these changes for now 👍

**Comment by hoangviet1993** [comment]
PTAL @seanlip

**Pull request merged**